tests/api/task1.test.js

```diff
@@ -0,0 +1,43 @@
+import {test, expect} from '@playwright/test';
+
+test.describe("API testing", () => {
+
+  const baseUrl = "http://localhost:3030/"
+
+  test("Get all Lists with their List Items", async ({request}) => {
+
+    const response = await request.get(`${baseUrl}/lists`)
+    expect(response.status()).toBe(200)
+
+    const responseBody = JSON.parse(await response.text())
+    console.log(responseBody)
+  })
+
+  test("Assert Invalid Endpoint for GET request", async ({request}) => {
+
+    const response = await request.get(`${baseUrl}/invalidendpoint`)
+    expect(response.status()).toBe(404)
+  })
+
+  test("Add a List with POST request", async ({request}) => {
+
+    const response = await request.post(`${baseUrl}/lists/add`, {
+      data: {
+        name: "Toptal"
+      },
+    })
+
+      const responseBody = JSON.parse(await response.text())
+    // console.log(responseBody)
```

Comment:
I recommend to start from now to take care of cleanless of the commits. It's a good practice to remove unused parts of code before comming the changes.